legion/engine/rendering/imgui_impl/imgui_widgets.cpp

```diff
@@ -4059,7 +4059,7 @@ bool ImGui::InputTextEx(const char* label, const char* hint, char* buf, int buf_
     if (g.ActiveId == id && !g.ActiveIdIsJustActivated && !clear_active_id)
     {
         IM_ASSERT(state != NULL);
-        IM_ASSERT(io.KeyMods == GetMergedKeyModFlags() && "Mismatching io.KeyCtrl/io.KeyShift/io.KeyAlt/io.KeySuper vs io.KeyMods"); // We rarely do this check, but if anything let's do it here.
+        //IM_ASSERT(io.KeyMods == GetMergedKeyModFlags() && "Mismatching io.KeyCtrl/io.KeyShift/io.KeyAlt/io.KeySuper vs io.KeyMods"); // We rarely do this check, but if anything let's do it here.
```

Comment:
I read the comment behind it saying that the check is rarely done. However can it truly be removed? and if it can shouldn't it be completely removed (not commented out) or put inside a #ifdef #endif

Reply:
i left a comment to explain it in the code now